scripts/presubmit.sh

```diff
@@ -12,7 +12,7 @@ find . -name "mock_*" -delete
 golint --set_exit_status ./...
 go vet ./...
 misspell -error -locale US .
-nolicense=$(find . -name "*.go" | xargs grep -L "Apache License")
+nolicense=$(find . -path ./protoc -prune -name "*.go" -o -name "*.proto" | xargs grep -L "Apache License")
```

Comment:
OK, here's the find incantation that I think works:

`find . -path ./protoc -prune -o -name '*.proto' -or -name '*.go' -print`